numba/cuda/tests/cudapy/test_inspect.py

```diff
@@ -148,6 +148,21 @@ def f(x):
 
         self.assertIn('nvdisasm is required', str(raises.exception))
 
+    @skip_without_nvdisasm('nvdisasm needed for inspect_cfg()')
+    def test_inspect_cfg(self):
```

Comment:
For consistency with the name of the method under test:

```suggestion
    @skip_without_nvdisasm('nvdisasm needed for inspect_sass_cfg()')
    def test_inspect_sass_cfg(self):
```